File: solutions/15/sol.js
const N = 20;

const getAnswer = (n) => {
  const arr = new Array(n + 1).fill(true).map(() => new Array(n + 1).fill(-1));
  for (let index = 0; index <= n; index++) {
    arr[0][index] = 1;
    arr[index][0] = 1;
  }
  for (let row = 1; row <= n; row++) {
    for (let col = 1; col <= n; col++) {
      arr[row][col] = arr[row - 1][col] + arr[row][col - 1];
    }
  }
  return arr[n][n];
}

const answer = getAnswer(N);
console.log(answer);

File: solutions/67/sol.go
package main

import (
	"fmt"
)

func getAnswer(arr [][]int) [][]int {
	n := len(arr)
	for i := 1; i < n; i++ {
		arr[i][0] += arr[i-1][0]
		arr[i][i] += arr[i-1][i-1]
	}
	return arr
}

func main() {
	arr := [][]int{
		{3},
		{7, 4},
		{2, 4, 6},
		{8, 5, 9, 3},
	}
	fmt.Println(getAnswer(arr))
}
